bin/update.js

```diff
@@ -0,0 +1,219 @@
+/* eslint-disable no-param-reassign */
+const cliProgress = require('cli-progress');
+
+const connection = require('../lib/ezunpaywall');
+
+const logger = require('../lib/logger');
+
+const createCliProgress = (percent, task, file) => {
+  const bar = new cliProgress.SingleBar({
+    format: `progress [{bar}] {percentage}% | {value}/{total} | ${task} - ${file}`,
+  });
+  bar.start(100, percent);
+  return bar;
+};
+
+/**
+ * get list of states in ezunpaywall
+ * @returns {array<String>} array of name of snapshot
+ */
+const getState = async (file, latest) => {
+  const ezunpaywall = await connection();
+  let res;
+  try {
+    res = await ezunpaywall({
+      method: 'POST',
+      url: `/api/update/state${file ? `/${file}` : ''}`,
+      params: {
+        latest,
+      },
+    });
+  } catch (err) {
+    logger.errorRequest('POST', err?.response?.config, err?.response?.status);
+    logger.error(err);
+    process.exit(1);
+  }
+  return res?.data?.state || res?.data;
+};
+
+const verbose = async () => {
+  let index = 0;
+  let state;
+  state = await getState('', true);
+  let { steps } = state;
+  steps.forEach(async (step) => {
+    if (step?.percent === 100) {
+      createCliProgress(100, step?.task, step?.file);
+      console.log('\r');
+      index += 1;
+    }
+  });
+
+  while (!state?.done) {
+    state = await getState('', true);
+    steps = state.steps;
+    steps = steps.filter((x) => x.task !== 'getChangefiles');
+    let latestStep = steps[index];
+
+    let bar;
+    if (latestStep?.task === 'insert' || latestStep?.task === 'download') {
+      bar = createCliProgress(latestStep?.percent, latestStep?.task, latestStep?.file);
+    }
+
+    if (latestStep?.status === 'inProgress') {
+      while (latestStep?.percent !== 100) {
+        if (state?.error) {
+          logger.error('process ended by error');
+          process.exit(1);
+        }
+        try {
+          state = await getState('', true);
+        } catch (err) {
+          logger.error('Cannot get state');
+          logger.error(err);
+        }
+        steps = state?.steps;
+        if (Array.isArray(steps)) {
+          steps = steps.filter((x) => x.task !== 'getChangefiles');
+          latestStep = steps[index];
+          bar.update(Number(latestStep.percent));
+        }
+        await new Promise((resolve) => setTimeout(resolve, 1000));
+      }
+      console.log('\r');
```

Comment:
```suggestion
      console.log();
```

`console.log()` already prints to `stdout` with a `new line`. Additionally, the proper character for line breaks is `\n` (`\r` is a `carriage return`).